.changeset/old-actors-learn.md

```diff
@@ -0,0 +1,11 @@
+---
+'create-astro': minor
+---
+
+Rework the experience of creating a new Astro project.
+
+- Update the list of templates to include Starlight and remove the minimal template.
```

Comment:
```suggestion
- Updates the list of templates to include Starlight and combines the "minimal" and "basics" templates into a new, refreshed "Basics" template to serve as the single, minimal Astro project starter.
```
Maybe something like this is fine here, and then we don't need the paragraph at the end?